temporian/core/event_set_ops.py

```diff
@@ -1501,6 +1501,38 @@ def abs(
 
         return abs(self)
 
+    def __round__(self):
+        from temporian.core.operators.unary import round
+
+        return round(input=self)
+
+    def round(
+        self: EventSetOrNode,
+    ) -> EventSetOrNode:
+        """Rounds the values of an [`EventSet`][temporian.EventSet]'s features to the nearest integer.
+
```

Comment:
Add another line specifying that only float types are allowed, and the output type will always be the same as the input's